environment.yml

```diff
@@ -78,12 +78,11 @@ dependencies:
   - black=22.10.0
   - cpplint
   - flake8=6.0.0
-  - flake8-bugbear=22.7.1 # used by flake8, find likely bugs
   - isort>=5.2.1  # check that imports are in the right order
   - mypy=0.991
   - pre-commit>=2.15.0
-  - pycodestyle  # used by flake8
   - pyupgrade
+  - ruff=0.0.205
```

Comment:
Needs to be synced with pre-commit (ideally enforced or comments in both files)

Reply:
Should ruff actually be here? People are supposed to run pre-commit.

Reply:
> Should ruff actually be here? People are supposed to run pre-commit.

I think we don't require people to use pre-commit (only recommend it in our contributing guide). In general the environment seems to contain all tools, it also includes black, isort, etc